attributes/default.rb

```diff
@@ -20,3 +20,4 @@
 default['build-essential']['compile_time'] = false
 default['build-essential']['mingw32']['path'] = "#{ENV['SYSTEMDRIVE']}\\mingw32"
```

Comment:
Lets get rid of these since we're only going to install `msys2` going forward. We can always add it back and have 4 different build toolchains on the boxes later 😄